library/alloc/src/collections/btree/borrow.rs

```diff
@@ -1,29 +1,29 @@
 use core::marker::PhantomData;
 use core::ptr::NonNull;
 
-/// A reborrow of some reference with lifetime `'t`, where the reborrow lifetime
-/// `'n` (see `DormantMutRef::new`) is too complicated for the compiler.
-/// In other words, the compiler cannot prove that `'n` ends before `awaken` is called.
-pub struct DormantMutRef<'t, T> {
+/// A reborrow of some reference with lifetime `'a`, where the reborrow lifetime
+/// `'b` (see `new`) is too complicated for the compiler. In other words, when
+/// the compiler cannot prove that `'b` ends before `awaken` is called.
```

Comment:
The second sentence is not a proper sentence, it's just a "when" clause.

Reply:
Apart from the first few words, I don't like the description much. It's mostly about what the compiler cannot do, and not what the subject allows. I'm still trying to understand what that actually is and will edit it in. I had to look up "IOW", and I don't think it was Isle Of Wight...

Reply:
Yeah sorry that was just something I quickly dumped into the playground^^ (IOW = in other words)